integration/oci_calls/oci_call_test.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will verify the arguments of
+# running or stopping a container matches
+# with the OCI calls
+
+set -e
+
+cidir=$(dirname "$0")
```

Comment:
this script is not on the ci directory, so I would recommend to change the name of the variable